modules/persistent-storage-local-pod.adoc

```diff
@@ -5,35 +5,36 @@
 [id="local-pod_{context}"]
 = Attach the local claim
```

Comment:
I think this title could be expanded for findability and scannability. It would be useful to say more about what is being attached to, in the title.

Reply:
I realise that this might be out of scope for this PR.

Reply:
Yeah, it is out of scope for now but I'll consider it for post-4.6. 😸